src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -179,6 +194,36 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
         /// ]]></format>
         /// </example>
         public static ValueMappingEstimator<TInputType, TOutputType> MapValue<TInputType, TOutputType>(
+            this TransformsCatalog.ConversionTransforms catalog,
+            IEnumerable<TInputType> keys,
+            IEnumerable<TOutputType> values,
```

Comment:
I don't like this API very much at all. If there really *is* a one to one correspondence between the two, why not just have it be a `System.Collections.KeyValuePair<TInputType, TOutputType>` and be done with it? Why parallel sequences? That doesn't make much sense to me.